src/transformers/file_utils.py

```diff
@@ -211,6 +211,7 @@
 TF_WEIGHTS_NAME = "model.ckpt"
 FLAX_WEIGHTS_NAME = "flax_model.msgpack"
 CONFIG_NAME = "config.json"
+FEATURE_EXTRACTOR_NAME = "feature_extractor_config.json"
```

Comment:
I think I would name this to a more generic `"preprocessor.json"`. feature_extractor makes little sense for vision.